vtr_flow/scripts/python_libs/doc/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
This should probably be part of the verilog-to-routing docs at https://github.com/verilog-to-routing/vtr-verilog-to-routing/tree/master/doc